code-challenges.test.js

```diff
@@ -45,12 +98,39 @@ const letterE = "e"
 
 // a) Create a test with expect statements using the variable provided.
 
+// psuedo code
+// conditonal statment to check if indexes equal each other i.e index 0 equals index 1 AND index 3 equals index 4
+
 const hand1 = [5, 5, 5, 3, 3]
 // Expected output: true
 const hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe("fullHouseChecker", () => {
+  it("Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.", () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    // Expected output: true
+    const hand2 = [5, 5, 3, 3, 4]
+    // Expected output: false
+    const hand3 = [5, 5, 5, 5, 4]
+    // Expected output: false
+    expect(fullHouseChecker(hand1)).toEqual(true)
+    expect(fullHouseChecker(hand2)).toEqual(false)
+    expect(fullHouseChecker(hand3)).toEqual(false)
+  })
+})
+
+//
+// console.log(fullHouseChecker(hand1));
+// console.log(fullHouseChecker(hand2));
+// console.log(fullHouseChecker(hand3));
 
 // b) Create the function that makes the test pass.
+
+const fullHouseChecker = (arr) => {
+    if((arr[0] === arr[1])&&(arr[3] === arr[4])) {
+        return true
+    }else return false
+}
```

Comment:
False positive testing. When there are so few tests we can accidentally make code with major bugs that will cause issues down the road. 

Consider:
```javascript
hand4 = [1,1,2,3,3]
```